static/src/javascripts/projects/common/modules/experiments/tests/commercial-consent-global.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+export const commercialConsentGlobal: ABTest = {
+    id: 'CommercialConsentGlobal',
+    start: '2019-03-25',
+    expiry: '2019-04-16',
+    author: 'Frankie Hammond',
+    description: 'Test the TCF globally',
+    audience: 0.02,
+    audienceOffset: 0.1,
+    successMeasure: 'TCF is served globally',
+    audienceCriteria: 'internal',
```

Comment:
Minor: is our audience `'internal'`? Our audience criteria is probably 'all users' or something else to indicate that this is live test. 😄 

https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/01-ab-testing.md